great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -233,6 +238,11 @@ def run(
         Returns:
             "RuleBasedProfilerResult" dataclass object, containing essential outputs of profiling.
         """
+        if include_column_names is not None and exclude_column_names is not None:
+            raise ValueError(
+                "You may either use `include_column_names` or `exclude_column_names` (but not both)."
+            )
```

Comment:
The check is added in both places because as a user, I should be able to use the functionality from both the RBP and the assistant.

Reply:
@cdkini I do not feel good about this approach, unless this is the only way forward.  Is there any chance we could not merge it and put our minds together?  One thing is to "sanitize/filter" the result.  An entirely another thing is to update the RBP run method. /cc @NathanFarmer -- thank you.